src/components/cards/CommentCard.tsx

```diff
@@ -0,0 +1,245 @@
+import React from "react";
+import { FontCSS, TdCSS } from "../../css";
+import { palette } from "@guardian/src-foundations";
+import { Content, Tag } from "../../api";
+import { formatImage } from "../../image";
+import sanitizeHtml from "sanitize-html";
+import { Table, RowCell, TableRowCell, TableRow } from "../../layout/Table";
+import { Headline } from "../../components/Headline";
+import { Image } from "../../components/Image";
+import { headline } from "../../styles/typography";
+
+type Size = "small" | "large";
+
+const tdStyle: TdCSS = {
+    backgroundColor: palette.neutral[100],
+    borderTop: `1px solid ${palette.opinion.main}`,
+    padding: "0"
+};
+
+const metaWrapperStyle = (size: Size): TdCSS => {
+    const rightPad = size === "large" ? "40px" : "10px";
+    const bottomPad = size === "large" ? "0" : "15px";
+    return {
+        padding: `3px ${rightPad} ${bottomPad} 10px`
+    };
+};
+
+const standfirstStyle: TdCSS = {
+    padding: "20px 10px 10px 10px",
+    verticalAlign: "bottom"
+};
+
+const linkStyle: FontCSS = {
+    textDecoration: "none"
+};
+
+const spanStyle: FontCSS = {
+    ...headline({ level: 1 }),
+    color: palette.neutral[7]
+};
+
+const columnStyleRight: TdCSS = {
+    width: "30%",
+    verticalAlign: "bottom",
+    padding: "0"
+};
+
+interface Props {
+    content: Content;
+    salt: string;
+    size: "large" | "small";
+    shouldShowImage?: boolean;
+    shouldShowProfileImage?: boolean;
+}
+
+const brazeParameter = "?##braze_utm##";
+
+const TrailText: React.FC<{
+    text: string;
+    linkURL: string;
+    size: Size;
+}> = ({ text, linkURL }) => {
+    return (
+        <td className="m-pad" style={standfirstStyle}>
+            <a style={linkStyle} href={linkURL}>
+                {" "}
+                <span style={spanStyle}>{text}</span>
+            </a>
+        </td>
+    );
+};
+
+// TODO add alt text
```

Comment:
Don't think this comment is needed - looks like the image already has a alt text